www/js/utils.js

```diff
@@ -31,6 +31,16 @@ function setInnerHtmlById(id, val) {
     document.getElementById(id).innerHTML = val;
 }
 
+function setHrefById(id, val) {
+    document.getElementById(id).href = val;
+}
+
+// Leads to the browser asking to open an installed wallet, but doesn't seem to trigger the Lightning Joule extension.
+// Lightning Joule extension is triggered when clicking on a proper link.
+function pay(elemendId){
```

Comment:
wird das noch irgendwo verwendet?

Reply:
eventuell reagiert die extension nur auf links mit entsprechenden hrefs und kann seitenaufrufe mit `lightning:...` nicht so auswerten, wie du's hier gedacht hast

Reply:
Nur in den auskommentierten `button`s - hab's entfernt